### ISSUE TITLE
Modern Warfare 2 splitscreen messed up.

### ISSUE THREAD

**Issue by gamrXerus** [RPCS3/rpcs3#10069]
## Quick summary
Please briefly describe what is not working correctly.
In a 4 player splitscreen match, player 1 also controls player 3, player 2 also controls player 4. Looking around is difficult and the screen sometimes get stuck for certain players. Its a bit difficult to explain.
## Details
Video showing problem: https://youtu.be/V48OBBlGVV0

Log: 
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/6260284/RPCS3.log.gz)

RSX capture: https://drive.google.com/file/d/1b4bJzGeT4BSq0lPFKuohbZcJw7bWaeBt/view?usp=sharing

#### 5. Please provide your system configuration:
OS: Windows 11 and Lubuntu 21.10
CPU: Intel Core i7 10700F
GPU: RX6800
Driver version:21.3.2 and mesa git with oibaf ppa
4 Logitech F310 controllers

**Comment by Megamouse** [comment]
Does this happen with both Vulkan and OpenGL renderers?

**Comment by gamrXerus** [comment]
> Does this happen with both Vulkan and OpenGL renderers?

Just tested it in OpenGL, yes it still happens

**Comment by gamrXerus** [comment]
@Megamouse So I did some more digging. I found a screenrecord of me playing with two other people just fine using parsec and the 0.0.11-10617 build. This was back when I had an NVIDIA GTX 1060. I downloaded it on Windows 10 and Ubuntu 21.04 but the issue was still there for both that build and the latest. Seems to only happen with 3 players or more. In 3 player mode i noticed the 1st player's screen was being rendered twice on the left hand side, player 2 was fine and player 3's screen was either stuck or switching from moving to stuck depending on how I moved my sticks.
Ubuntu 21.04 + oibaf ppa Vulkan
![bug3](https://user-images.githubusercontent.com/19245344/113787340-d68c8d00-9708-11eb-9421-362b6889d01f.jpg)
Ubuntu 21.04 + oibaf ppa OpenGL
![bug4](https://user-images.githubusercontent.com/19245344/113787449-0cca0c80-9709-11eb-9feb-4a9725c80461.jpg)
